src/readbin/ground_truth.ml

```diff
@@ -0,0 +1,26 @@
+open Bap.Std
+open Core_kernel.Std
+open Or_error
+
+
+let from_unstripped_bin bin : Addr.Set.t t =
+  let tmp = Filename.temp_file "bw_" ".symbol" in
+  let cmd = Printf.sprintf "bap-byteweight dump -i %s %S > %S" "symbols"
```

Comment:
just a small tip. Core_kernel brings `sprintf` and `printf` into scope, so even do not need to open `Printf` or `Format`, just use `sprintf` and `printf`.